datahub-frontend/conf/application.conf

```diff
@@ -30,6 +30,12 @@ play.modules.enabled += "auth.AuthModule"
 play.allowHttpContext = true
 play.allowGlobalApplication = true
 
+# Uncomment to override server provider and set the max header count to a higher value
+# This is useful while using proxies like Envoy that result in the frontend server rejecting GMS
+# responses as there's more than the max of 64 allowed headers
+#play.server.provider = server.CustomAkkaHttpServerProvider
```

Comment:
Is there any issue with making this the default? Looks like it will just fall back to the default settings if it doesn't find any custom configurations so this should be okay to leave uncommented. 

Typically users won't directly modify this file unless they're forking DataHub, so if it doesn't make sense to have as a default would prefer this is in a way that it can either be turned on/off with an env variable like:

play.server.provider = ${DATAHUB_AKKA_SERVER_PROVIDER_CLASS:server.DefaultAkkaHttpServerProvider}

but I think it really should be okay to just have it uncommented 😄

Reply:
Yeah, I think we can make this the default coupled with the fallback value like you suggested below.